### DISCUSSION TITLE
0.4.3

### DISCUSSION THREAD

**Discussion by kubilus1** [kubilus1/autoortho#131] [Announcements]
Several tweaks, including better backup chunk replacement for fewer pixelated replacement chunks.  Also going to try adding a non-self extracting zip to possibly get around Windows virus scanning issues.

## What's Changed
* Fixed readme to make it more up to date by @Simmer1234569 in https://github.com/kubilus1/autoortho/pull/111
* Check cache for best backup chunk.   Use websocket with flighttrack. by @kubilus1 in https://github.com/kubilus1/autoortho/pull/125

## New Contributors
* @Simmer1234569 made their first contribution in https://github.com/kubilus1/autoortho/pull/111

**Full Changelog**: https://github.com/kubilus1/autoortho/compare/0.4.2...0.4.3

<hr /><em>This discussion was created from the release <a href='https://github.com/kubilus1/autoortho/releases/tag/0.4.3'>0.4.3</a>.</em>

**Comment by h-a-l-t-e-r** [comment]
ERROR:fuse:Uncaught exception from FUSE operation read, returning errno.EINVAL.
Traceback (most recent call last):
  File "/tmp/onefile_1146870_1684022851_153447/refuse/high.py", line 863, in _wrapper
  File "/tmp/onefile_1146870_1684022851_153447/refuse/high.py", line 974, in read
  File "/tmp/onefile_1146870_1684022851_153447/refuse/high.py", line 1222, in __call__
  File "/tmp/onefile_1146870_1684022851_153447/autoortho_fuse.py", line 345, in read
  File "/tmp/onefile_1146870_1684022851_153447/getortho.py", line 638, in read_dds_bytes
  File "/tmp/onefile_1146870_1684022851_153447/getortho.py", line 585, in get_bytes
  File "/tmp/onefile_1146870_1684022851_153447/getortho.py", line 80, in wrapped
  File "/tmp/onefile_1146870_1684022851_153447/getortho.py", line 760, in get_img
  File "/tmp/onefile_1146870_1684022851_153447/getortho.py", line 829, in get_best_chunk
AttributeError: 'NoneType' object has no attribute 'crop'

This error occur if i start AO v0.4.3 with my cache folder from AO v0.4.2

**Reply by kubilus1** [reply]
XP11? XP12?  Windows, looks like, what ver?  Dokan or WinFSP?

**Reply by kubilus1** [reply]
I made an update that you can try again to see if this resolves the issue.

**Reply by h-a-l-t-e-r** [reply]
> XP11? XP12? Windows, looks like, what ver? Dokan or WinFSP?

AMD Ryzen 5 5600X, 32GB RAM, AMD Radeon RX6700XT
Ubuntu 22.04 LTS, Kernel 6.2.14, Mesa Vulkan drivers 23.0.3~kisak1~j

XP 12.05r1

**Reply by h-a-l-t-e-r** [reply]
> I made an update that you can try again to see if this resolves the issue.

Thank you for the new bin. I restored my 0.4.2 cache folder and it load OK.

**Comment by h-a-l-t-e-r** [comment]
There is a procedure to invalidate some old BI textures? My home airport got new images from BI last week. AO continue using the textures in the cache.

As a workaround i start AO with a new cache folder to get the new textures and then replace this new files on the old cache folder and restore it.

**Comment by Uljos** [comment]
Was trying to download autoortho 0.4.3-1 for Windows, got a Trojan alert from defender, is it a false positive so I can switch off defender to install?

**Reply by lknfly1** [reply]
Same here, I tried to download the 4.3 and got an alert from defender?

I'd like to know what does it mean or same question a false positive? I don't have defender activated full functions so what do to?

**Reply by lknfly1** [reply]
Trojan:Win32/Wacatac.H!ml

I'm kind of worried now about what I've read on this Trojan online so far. Two dudes got affected, one got is password email adress stolen, so I don't know... I didn't download it finally

**Comment by h-a-l-t-e-r** [comment]
Hi @kubilus1, can you add a '-v' option or a message on the console to know the version of the '.bin' file we are running?

**Comment by Wind58** [comment]
Hello
I've just downloaded auto ortho but can't use it. In setup tab everything looks ok but in scenery tab I get nothing; a black window only. No information at all. What am I doing wrong?
Manjaro Linux, Kernel 6.3.3-1
![ao](https://github.com/kubilus1/autoortho/assets/134738809/f752a434-964b-4e54-8a3c-f6ab852c7d3b)
[autoortho.log](https://github.com/kubilus1/autoortho/files/11579972/autoortho.log)

Edit:
Some more info from terminal
--
~  ./autoortho_lin.bin                                                    ✔ 
Config file found /home/cbs/.autoortho reading...
Saving config ... 
Wrote config file: /home/cbs/.autoortho
INFO:downloader:Looking for regions ...
INFO:downloader:Check for updates ...
WARNING:downloader:Couldn't update release info <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)>.
INFO:downloader:Using scenery dir /home/cbs/X-Plane 11/Custom Scenery
Updating config.
Saving config ... 
INFO:aoconfig:Wrote config file: /home/cbs/.autoortho
Wrote config file: /home/cbs/.autoortho
Config file found /home/cbs/.autoortho reading...
INFO:aoconfig:Config file found /home/cbs/.autoortho reading...
Quiting ...
Exiting ...

**Reply by CraigRat** [reply]
Getting the same for XP12: Arch kernel 6.3.5
Wrote config file: /home/craigrat/.autoortho

INFO:downloader:Looking for regions ...
INFO:downloader:Check for updates ...

WARNING:downloader:Couldn't update release info <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)>.

INFO:downloader:Using scenery dir /mnt/games/X-Plane 12/Custom Scenery

Updating config.

Saving config ...

**Reply by Pist0lenPeter** [reply]
Same here:

./autoortho_lin.bin
Config file found /home/peter/.autoortho reading...
Saving config ... 
Wrote config file: /home/peter/.autoortho
INFO:downloader:Looking for regions ...
INFO:downloader:Check for updates ...
WARNING:downloader:Couldn't update release info <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)>.
INFO:downloader:Using scenery dir /media/games/SteamLibrary/steamapps/common/X-Plane 11/Custom Scenery

**Comment by gabrielcassis** [comment]
Q: Once RUN is clicked (running on Windows 10), the log entry "WARNING... Websocket transport not available. Install simple-websocket for improved performance." appears. I also get that message despite having installed simple-websocket from python and enabled websocket in IIS. I can't find any documentation on how to configure AutoOrtho to use websocket. How do I resolve this warning?

**Comment by ghost** [comment]
I keep getting this "Errno 13 Permission denied" error message whenever the program extracts the first folder of the selected scenery pack i. Any quick help would be appreciated.

**Reply by ghost** [reply]
![image thumb jpeg 811af0376840ae11fd61ac745ebf997d](https://github.com/kubilus1/autoortho/assets/137732508/39eb70c9-c305-436e-afd5-ad87544a1d35)
